next.config.js

```diff
@@ -20,6 +20,16 @@ const nextConfig = {
         hostname: "images.unsplash.com",
         port: "",
       },
+      {
+        protocol: "https",
+        hostname: "unsplash.com",
+        port: "",
+      },
+      {
+        protocol: "https",
+        hostname: "de.wikipedia.org",
```

Comment:
I had the same mistake with wikipedia. The error code suggested using: `hostname: "upload.wikimedia.org",` instead, that worked for me.